### ISSUE TITLE
On verification of the QR, it says 'this qr code does not contain a corona proof'

### ISSUE THREAD

**Issue by barbetb** [minvws/nl-covid19-coronatester-ctcl-core#11]
On verification of the QR, it says 'this qr code does not contain a corona proof'.

Possibly the key is not generated correctly?

**Comment by confiks** [comment]
Hi barbetb! This repository is from March this year, and predates the existence of verifiable test and vaccination certificates. If you're looking for more information on the current implementation of European DCCs, [this repository](https://github.com/ehn-dcc-development/hcert-spec) is a good start. It links to sample implementations. The Netherlands uses [this production implementation](https://github.com/minvws/nl-covid19-coronacheck-hcert).

If you're looking for more information on the privacy-friendly Dutch QR codes, [this document](https://github.com/minvws/nl-covid19-coronacheck-app-coordination/blob/main/architecture/Privacy%20Preserving%20Green%20Card.md) is a good readup which explains the transition from test to vaccination certificates, and how privacy is maintained. The production implementation can be found in the [coronacheck-idemix](https://github.com/minvws/nl-covid19-coronacheck-idemix) and [coronacheck-mobile-core](https://github.com/minvws/nl-covid19-coronacheck-mobile-core) repository.

**Comment by joudrodiak** [comment]
barbetb is asking whether there is something wrong with implementing the QR code, hence the fact when it is scanned, it is not identified.
So what is going wrong?

**Comment by confiks** [comment]
I don't quite follow you. Can you rephrase? Dutch would also be okay.

**Comment by joudrodiak** [comment]
When the generated QR code is scanned with coronacheck scanner app, it gives an error that the QR code is not recognized rather than giving the persons info.

**Comment by confiks** [comment]
Ah, right. That's expected. QRs generated with this project are not compatible and can't be used with the CoronaCheck scanner app. The CoronaCheck scanner only accepts QR codes that were created via official routes via https://coronacheck.nl, the GGD and general practitioners.

If you want to make compatible QR codes that can be scanned with you own version of the CoronaCheck Scanner app, you need to check out the projects I linked above.

**Comment by joudrodiak** [comment]
Yeah, that is the answer to barbetb I reckon.
